assets/content/less/buttons/buttons.less

```diff
@@ -0,0 +1,73 @@
+// lesshint qualifyingElement: { enabled: true, allowWithClass: true, allowWithAttribute: true }
+.btn {
+    font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+    font-size: 14px;
+    padding-left: 29px;
+    padding-right: 31px;
+
+    &:hover {
+        box-shadow: 0px 2px 4px 0px rgba(0, 0, 0, 0.23);
+    }
+
+    // Blue button
+    &.btn-primary,
+    &.btn-primary:hover {
+        background: #65a6de;
```

Comment:
Also, I dont know if you are using bootstrap here, I didnt find the dependency in the bower file, but if you do, then you can simply modify the bootstrap variable value for each button. That will be much cleaner.